jupyterlab_git/_version.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Project Jupyter.
 # Distributed under the terms of the Modified BSD License.
 
-version_info = (0, 22, 3)
+version_info = (0, 23, 0)
```

Comment:
```suggestion
version_info = (0, 30, 0)
```

Reply:
why jump from 22 to 30?

Reply:
When the extension was ported to JupyterLab 2, it was decided to jump from 0.10 to 0.20 to left version available for version with backported fix and features. So it will be good to do the same with JLab 3 to left version 0.2x available for backports for JupyterLab 2.